### DISCUSSION TITLE
running webpack-dev-server --open results in error

### DISCUSSION THREAD

**Discussion by donovanvanheerden** [webpack/webpack#14983] [Q&A]
Running `webpack-dev-server --hot` with `open:true` in the devServer options results in an error being thrown when trying to spawn child process. Not sure if I have a weird environment, but I checked all my environment variables and this specific issue seems to be that it does not recognise `%SystemRoot%`, since that is what is undefined when trying to spawn child process, see below.

![image](https://user-images.githubusercontent.com/13114346/146165642-dc91c43c-e90f-4de1-8ed8-b85c1f354d73.png)

Node: 16.13.0
NPM: 8.1.0
Windows: 21H2

![image](https://user-images.githubusercontent.com/13114346/146172513-f204e823-176a-453f-b7c6-24707d88dc95.png)

This doesn't appear to be the problem when I use the git bash terminal. Does anyone know how to fix this for `cmd` and `powershell` terminals?

**Comment by alexander-akait** [comment]
I think bug in https://github.com/sindresorhus/open, feel free to report

**Reply by donovanvanheerden** [reply]
Before I do that, I decided to try the new `create-react-app` since it now uses webpack v5, and I don't get an issue there when using any terminal. I guess I need to eject and compare configs, maybe I missed something.

**Reply by donovanvanheerden** [reply]
I see my mistake, I took over the config and saw there was some manipulation of `process.env` happening which resulted in `SystemRoot` being lost for `cmd` and `powershell`. Weird this didn't cause problems for `git bash` though.
